src/lib/spinel/radio_spinel.hpp

```diff
@@ -158,19 +158,10 @@ class RadioSpinel : private Logger
     /**
      * Initialize this radio transceiver.
      *
-     * @param[in]  aSpinelInterface            A reference to the Spinel interface.
-     * @param[in]  aResetRadio                 TRUE to reset on init, FALSE to not reset on init.
      * @param[in]  aSkipRcpCompatibilityCheck  TRUE to skip RCP compatibility check, FALSE to perform the check.
-     * @param[in]  aIidList                    A Pointer to the list of IIDs to receive spinel frame from.
-     *                                         First entry must be the IID of the Host Application.
-     * @param[in]  aIidListLength              The Length of the @p aIidList.
      *
      */
-    void Init(SpinelInterface    &aSpinelInterface,
-              bool                aResetRadio,
-              bool                aSkipRcpCompatibilityCheck,
-              const spinel_iid_t *aIidList,
-              uint8_t             aIidListLength);
+    void Init(bool aSkipRcpCompatibilityCheck, bool aSoftwareReset, SpinelDriver *aSpinelDriver);
```

Comment:
Document `aSoftwareReset` and `aSpinelDriver`.

Reply:
Updated.